### ISSUE TITLE
Support keyword 'all' to represent all ports?

### ISSUE THREAD

**Issue by jeking3** [online-labs/port-range#2]
It would be nice to have an "all ports" optional mechanism in port-range where it would match anything and the string representation would be "all".  Thoughts?

**Comment by kdeldycke** [comment]
Oh I see. You'd like to have a kind of global constant called `port_range.ALL` so you can use that to define clean defaults, like to check overlapping ranges as you suggested at https://github.com/scaleway/port-range/issues/3 right?

**Comment by jeking3** [comment]
Correct, it would make a good default in some situations like serialization.
